spec/Section 3 -- Type System.md

```diff
@@ -835,6 +838,9 @@ of rules must be adhered to by every Object type in a GraphQL schema.
       3. The object field may include additional arguments not defined in the
          interface field, but any additional argument must not be required, e.g.
          must not be of a non-nullable type.
+5. If an object type declares that it implements one or more interfaces that
+   implement other interfaces, it must also include each transitively
+   implemented interface in its list of implemented interfaces.
```

Comment:
This should be part of step 4 above. I think it can be stated more simply in a way that does not require transitive traversal (eg if each "implements" is valid, then they're all valid)